tests/packages/packages/d/node_modules/c/node_modules/b/assembly/b.ts

```diff
@@ -0,0 +1,9 @@
+import {A} from "a";
```

Comment:
nit: `{ A }`, and indentation is off (seems this is the case in others files as well)

Reply:
Yeah this is why I wanted the symbolic links, I fixed the first one, but forgot about the copies.